Formula/red.rb

```diff
@@ -0,0 +1,16 @@
+class Red < Formula
+  desc "Next-gen programming language, strongly inspired by REBOL"
+  homepage "http://www.red-lang.org"
+  url "http://static.red-lang.org/dl/mac/red-061"
+  sha256 "afefaa392e5dbc1ec6d8805376ecffe86a1f6d1ce46d426800556f3c4f698693"
+  head "https://github.com/red/red.git"
+
+  def install
+    mv "red-061", "red"
+    bin.install "red"
+  end
+
+  test do
+    system "#{bin}/red", "--version"
```

Comment:
Thanks for the advice, I will do a look at cmake.rb example. But I am also waiting for your answer (see first comment) about the binary installation instead of source installation. Thanks a lot for this  excellent installation system.